backend/routes/Quests.go

```diff
@@ -0,0 +1,36 @@
+package routes
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/backend/core/" // This points to your core folder
+)
+
+// GetQuests handler and also retrieves all quests from the database
+func GetQuests(w http.ResponseWriter, r *http.Request) {
```

Comment:
Changing this to 2 separate `GetDailyQuests` and `GetMainQuests` functions and routes would be awesome, if you can.